packages/jest-message-util/src/index.js

```diff
@@ -227,7 +227,7 @@ export const formatStackTrace = (
   if (topFrame) {
     const filename = topFrame.file;
 
-    if (path.isAbsolute(filename)) {
+    if (path.isAbsolute(filename) && fs.existsSync(filename)) {
```

Comment:
Would it be better to just catch the `readFileSync` to save an io operation?

Reply:
What we should do is check the hastefs cache and just read from that

Reply:
See https://github.com/facebook/jest/pull/5087#discussion_r157170622. Fixing that (ignoring files not in the cache) is the real fix

Reply:
@SimenB I don't know how HasteFS works or how to read its cache, however I suspect there could be cases where the file would not appear in it either. One of those cases could be running `vm.runInThisContext` or `vm.runInNewContext` from within a test, and the contents of the file is evaled (rather than being read from the disk). Unless that somehow ends up in the HasteFS too? I'm only mentioning it, since it's also exactly what happens in the `webpack` case.

Could I try/catch the `readFileSync` for now, so you could ship this bugfix as soon as possible, and we can prepare the proper fix at a later time, since I don't have enough knowledge about HasteFS? Unless it's something really simple, and you could point me to the right place?

Cheers!

Reply:
It's not a simple change as we have to somehow thread the cache all the way through.

Sticking the `readFileSync` in a `try` should be good enough